project_template/tasks.py

```diff
@@ -270,11 +272,42 @@ class TaskExtraValuableTable(CountTableRowsTask):
     child_class = TaskExtraValuableRowEntry
 
 
+class TaskOwnedIncomeFromDeferredUseOfGoodsRowEntry(DigitalizationTask):
+    task_form = forms.TranscribeOwnedIncomeFromDeferredUseOfGoodsRowEntry
+    template_name = "tasks/owned_income_from_deferred_use_of_goods.html"
+
+    def save_verified_data(self, verified_data):
+        holder_person, _ = models.Person.objects.get_or_create(
+            name=verified_data['holder_name'],
+            surname=verified_data['holder_surname'])
+
+        source_of_goods_source_name = verified_data.get('source_of_goods_source_name')
+        source_of_goods_source_surname = verified_data.get('source_of_goods_source_surname')
+        source_person = None
+        if source_of_goods_source_name and source_of_goods_source_surname:
+            source_person, _ = models.Person.objects.get_or_create(
+            name=source_of_goods_source_name,
+            surname=source_of_goods_source_surname)
+
+        owned_deferred_use_of_goods, _ = models.OwnedIncomeFromDeferredUseOfGoodsTableEntry.objects.get_or_create(
+            holder_relationship=verified_data['holder_relationship'],
+            holder_person=holder_person,
+            source_person=source_person,
+            source_of_goods=verified_data.get('source_of_goods_source_institution'),
```

Comment:
Why are you accessing some dictionary values with `.get()` and some by key? If you access a dictionary value using `.get()`, you will receive the default value (`None`) if the key isn't present in the dictionary. This masks possible errors. Can you explain why you made this choice?